topmenubar/src/main/java/de/interseroh/tmb/client/common/CopyTxtcolorToToggleNativeJavaScript.java

```diff
@@ -0,0 +1,16 @@
+package de.interseroh.tmb.client.common;
+
+/**
+ * Created by volker on 06.05.2017.
+ */
+public class CopyTxtcolorToToggleNativeJavaScript {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1118)